### PR TITLE
Action dependencies

### PR DESCRIPTION
**Pull request by trav3711**
Addressing dependency action bugs

### CODE REVIEW

File: tools/pip_install.sh [hunk 1]
```diff
@@ -89,4 +89,5 @@ ontology_validator_setup
 instance_validator_setup
 explorer_setup
 guid_generator_setup
+abel_setup
```
Comment (charbull) [inline tools/pip_install.sh:92]: does this add the pip in the abel setup?

File: .github/workflows/tools.yml [hunk 2]
```diff
@@ -138,7 +138,7 @@ jobs:
       run: |
         pylint --rcfile ./styles/pylintrc ./tools/explorer
     - name: Ontology Explorer Tests
-      if: steps.changes.outputs.explorer == 'true'
+      if: steps.changes.outputs.explorer == 'true' OR steps.changes.outputs.type_validator == 'true' OR steps.changes.outputs.instance_validator == 'true'
```
Comment (charbull) [inline .github/workflows/tools.yml:141]: does the explorer relies on the instance validator? I dont think so

File: .github/workflows/tools.yml [hunk 3]
```diff
@@ -73,7 +73,7 @@ jobs:
       run: |
         pylint --rcfile ./styles/pylintrc ./tools/validators/instance_validator
     - name: Instance Yaml Validator Tests
-      if: steps.changes.outputs.instance_validator == 'true'
+      if: steps.changes.outputs.instance_validator == 'true' OR steps.changes.outputs.type_validator == 'true'
```
Comment (charbull) [inline .github/workflows/tools.yml:76]: the or is not supported, please use the expression here: https://docs.github.com/en/actions/learn-github-actions/expressions#operators

File: .github/workflows/tools.yml [hunk 4]
```diff
@@ -89,7 +89,7 @@ jobs:
       run: |
         pylint --rcfile ./styles/pylintrc ./tools/abel
     - name: ABEL Tests
-      if: steps.changes.outputs.abel == 'true'
+      if: steps.changes.outputs.abel == 'true' OR steps.changes.outputs.instance_validator == 'true' OR steps.changes.outputs.type_validator == 'true'
```
Comment (charbull) [inline .github/workflows/tools.yml:92]: the or is not supported, please use the expression here: https://docs.github.com/en/actions/learn-github-actions/expressions#operators

File: .github/workflows/tools.yml [hunk 5]
```diff
@@ -105,7 +105,7 @@ jobs:
       run: |
         pylint --rcfile ./styles/pylintrc ./tools/guid_generator
     - name:  GUID Generator Tests
-      if: steps.changes.outputs.guid_generator == 'true'
+      if: steps.changes.outputs.guid_generator == 'true' OR steps.changes.outputs.instance_validator == 'true'
```
Comment (charbull) [inline .github/workflows/tools.yml:108]: the or is not supported, please use the expression here: https://docs.github.com/en/actions/learn-github-actions/expressions#operators

File: .github/workflows/tools.yml [hunk 6]
```diff
@@ -154,7 +154,7 @@ jobs:
       run: |
         pylint --rcfile ./styles/pylintrc ./tools/scoring
     - name: Configuration Scorer Tests
-      if: steps.changes.outputs.scoring == 'true'
+      if: steps.changes.outputs.scoring == 'true' OR steps.changes.outputs.instance_validator == 'true'
```
Comment (charbull) [inline .github/workflows/tools.yml:157]: the or is not supported, please use the expression here: https://docs.github.com/en/actions/learn-github-actions/expressions#operators

**Review by charbull — CHANGES_REQUESTED** [review]

File: tools/pip_install.sh (hunk 1 above)
Comment (trav3711) [reply tools/pip_install.sh:92]: Yeah I don't think anymore needs to be added. I checked in the instance validator, for example, and there aren't extra files besides `setup.py`. `python -m pip install .` is alaready defined in the `abel_setup()` function.

File: .github/workflows/tools.yml (hunk 2 above)
Comment (trav3711) [reply .github/workflows/tools.yml:141]: `model_test.py` and `ontology_wrapper_test.py` rely on instance validator's simplified universe.

File: .github/workflows/tools.yml (hunk 3 above)
Comment (trav3711) [reply .github/workflows/tools.yml]: Done.

File: .github/workflows/tools.yml (hunk 4 above)
Comment (trav3711) [reply .github/workflows/tools.yml]: Done.

File: .github/workflows/tools.yml (hunk 5 above)
Comment (trav3711) [reply .github/workflows/tools.yml]: Done.

File: .github/workflows/tools.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (trav3711) [reply .github/workflows/tools.yml]: Done.

File: .github/workflows/tools.yml (hunk 6 above)
Comment (trav3711) [reply .github/workflows/tools.yml]: Done.

**Review by charbull — APPROVED** [review]

**Pull request merged**